File: mega-zep-frontend/src/main/angular/frontend/src/app/modules/shared/models/MonthlyReport/JourneyWarning.ts

export class JourneyWarning {

  private _date: Date;
  private _warnings: Array<String>;

  get date(): Date {
    return this._date;
  }

  set date(value: Date) {
    this._date = value;
  }

  get warnings(): Array<String> {
    return this._warnings;
  }

  set warnings(value: Array<String>) {
    this._warnings = value;
  }
}

File: mega-zep-backend/src/main/java/com/gepardec/mega/monthlyreport/journey/JourneyWarning.java
package com.gepardec.mega.monthlyreport.journey;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.ArrayList;
import java.util.List;

@NoArgsConstructor
@Data
public class JourneyWarning {
    @JsonDeserialize(using = LocalDateDeserializer.class)
    @JsonSerialize(using = LocalDateSerializer.class)
    private LocalDate date;
    private List<String> warnings = new ArrayList<>(0);
}